src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -337,7 +337,7 @@ struct WgVertexBufferInd {
         float cosine = dot(nrm, nrm1);
         float angle = std::acos(dot(nrm1, -nrm2));
         float miterRatio = 1.0f / (std::sin(angle) * 0.5f);
-        if (miterRatio <= miterLimit) {
+        if ((miterRatio <= miterLimit) && (!tvg::zero(cosine)) && (!tvg::zero(angle))) {
```

Comment:
I meant after l.338 to avoid division by 0 while miterRatio is calculated

I confirm, that now the error is gone

Reply:
Lets keep it as is... I dont want to overcomlicate the logic in this place